01-node-tutorial/answers/writeWithPromisesThen.js

```diff
@@ -0,0 +1,19 @@
+const { writeFile, readFile } = require("fs").promises;
```

Comment:
You imported `readFile` but never used it. Add the logic to read the file using it or remove this unused import.